ci-plutus-benchmark.sh

```diff
@@ -0,0 +1,44 @@
+#!/usr/bin/env bash
+
+# ci-plutus-benchmark: Run benchmarks on 2 branches, compare the results and
+# add a comment on the corresponding PR on GitHub.
+#
+# This script is supposed to be executed by https://buildkite.com/input-output-hk/plutus-benchmark
+# which itself is triggered through the GitHub workflow ./.github/workflows/benchmark.yaml
+#
+# USAGE: 
+# In order to trigger benchmarking for an open PR, add `/benchmark` as a comment to
+# the PR. The command will be acknowledged with a :rocket: reaction and when done a bot will
+# add the results as comment to the same PR.
+#
+# NOTE:
+# In order to post a comment to GitHub the buildkite runner needs to provide a
+# GitHub token with permissions to add comments in '/run/keys/buildkite-github-token'
+# otherwise this will fail.
+
+set -e
+
+if [ -z "$PR_NUMBER" ] ; then
+   echo "[ci-plutus-benchmark]: 'PR_NUMBER' is not set! Exiting"
+   exit 1
+fi
+echo "[ci-plutus-benchmark]: Processing benchmark comparison for PR $PR_NUMBER"
+
+echo "[ci-plutus-benchmark]: Updating cabal database ..."
+cabal update
+
+echo "[ci-plutus-benchmark]: Running benchmark for PR branch ..."
+cabal bench plutus-benchmark:validation >bench-PR.log 2>&1
```

Comment:
Hmm, we should probably run this inside a `nix-shell` too!

Reply:
The whole script is being executed inside a nix-shell though:

```
steps:
  - command:
    - "nix-shell --run './ci-plutus-benchmark.sh'"   
    agents:
      queue: "plutus"
```

So this should be fine right?